src/TestPrograms/OpenGL.hs

```diff
@@ -0,0 +1,250 @@
+{-# LANGUAGE OverloadedStrings, PatternSynonyms #-}
```

Comment:
I think I"m good leaving this demo in the core library since the `OpenGLRaw` dependency is opt-in via a flag. Maybe adding a link to the [original](https://github.com/IngwiePhoenix/FLTK/blob/master/test/cube.cxx) is a good idea? Since it's here do you think I should remove this demo from the [fltkhs-gl-demos](https://github.com/deech/fltkhs-gl-demos/blob/master/src/cube.hs)?

Reply:
I think it's okay to keep the demo in both places, since they might diverge anyways (like if we want a simpler four different example in the core package).

Relatedly, we should add that repo, and the altimeter one too, to the list of Demos in the readme (I had to find it by skimming through your GitHub repos :-)

I can throw that into this commit, if you don't mind.

Reply:
Haha, I would say let's hold off on the altimeter demo for now. It hasn't been updated in a while and requires a bunch of love. :)